src/Microsoft.DotNet.Interactive.Http.Tests/HttpKernelTests.cs

```diff
@@ -2311,6 +2311,73 @@ public async Task responses_to_named_requests_can_be_accessed_as_symbols_in_late
         secondResult.Events.Should().NotContainErrors();
     }
 
+    [Fact]
+    public async Task Response_headers_for_named_requests_can_be_accessed_correctly()
+    {
+
+        var responseHandler = new InterceptingHttpMessageHandler((message, _) =>
+        {
+            var response = new HttpResponseMessage(HttpStatusCode.OK);
+            response.RequestMessage = message;
+            var contentString = @"
```

Comment:
nit: A `"""` string literal may be better here to avoid escaping double quotes etc.